scripts/rollup/bundles.js

```diff
@@ -1018,13 +1041,17 @@ function getOriginalFilename(bundle, bundleType) {
       return `${name}.production.min.js`;
     case NODE_PROFILING:
       return `${name}.profiling.min.js`;
+    case IIFE_PROD:
+      return `${name}.production.min.js`;
```

Comment:
Similarly here, I don't think we should bother with the `.production.min.js` stuff. We do that in our packages that are consumed by a bundler (like ES Modules or Common JS) but in this case there is no bundler, Fizz is going to inject this directly.

So let's just output the exact filename.